netebpfext/user/fwp_um.cpp

```diff
@@ -813,6 +829,10 @@ _IRQL_requires_max_(DISPATCH_LEVEL) NTSTATUS NTAPI FwpsAcquireWritableLayerDataP
 _IRQL_requires_max_(DISPATCH_LEVEL) NTSTATUS NTAPI
     FwpsAcquireClassifyHandle0(_In_ void* classifyContext, _In_ UINT32 flags, _Out_ UINT64* classifyHandle)
 {
+    if (ebpf_fault_injection_inject_fault()) {
+        return STATUS_NO_MEMORY;
```

Comment:
Does WFP actually do a memory allocation in this API?
https://learn.microsoft.com/en-us/windows-hardware/drivers/ddi/fwpsk/nf-fwpsk-fwpsacquireclassifyhandle0 doesn't mention STATUS_NO_MEMORY, should it?

Reply:
Yes, from the source code of this API, it does allocate.
'This function allocates a WFP_CLASSIFY_CONTEXT_ALLOC object, shallow-
   copies all classify parameters and return an opaque handle to the
   caller.'